dev-infrastructure/global-pipeline.yaml

```diff
@@ -2,49 +2,15 @@ $schema: "pipeline.schema.v1"
 serviceGroup: Microsoft.Azure.ARO.HCP.Global
 rolloutName: Global Resource Rollout
 resourceGroups:
-- name: {{ .global.rg }}
-  subscription: {{ .global.subscription }}
-  steps:
-  # creates global infra
-  # * the parent DNS zones for the ARO HCP services
-  # * the global Grafana instance
-  - name: global-infra
-    action: ARM
-    template: templates/global-infra.bicep
-    parameters: configurations/global-infra.tmpl.bicepparam
-    deploymentLevel: ResourceGroup
-  # creates DNS delegation for the ARO HCP global SVC zone
-  - name: svcChildZone
-    action: DelegateChildZone
-    parentZone:
-      configRef: dns.parentZoneName
-    childZone:
-      configRef: dns.svcParentZoneName
-    dependsOn:
-    - global-infra
-  # creates DNS delegation for the ARO HCP global CX zone
-  - name: cxChildZone
-    action: DelegateChildZone
-    parentZone:
-      configRef: dns.parentZoneName
-    childZone:
-      configRef: dns.cxParentZoneName
-    dependsOn:
-    - global-infra
-  # create global ARO HCP ACRs for OCP and SVC images
-  - name: global-acrs
-    action: ARM
-    template: templates/global-acr.bicep
-    parameters: configurations/global-acr.tmpl.bicepparam
-    deploymentLevel: ResourceGroup
 - name: {{ .imageSync.rg }}
   subscription: {{ .global.subscription }}
   steps:
   # deploys the image mirror for the ACRs
-  - name: imagemirror
-    action: ARM
-    template: templates/global-image-sync.bicep
-    parameters: configurations/global-image-sync.tmpl.bicepparam
-    deploymentLevel: ResourceGroup
-    dependsOn:
-    - global-acrs
+  - name: decrypt secrets
```

Comment:
i'm not sure right now if whitespaces are allowed as step names in EV2. i would keep the pattern of not using them and even enforcing it in the schema

naming suggestion - "decrypt-and-ingest-secrets"

Reply:
Was wondering about that :)